pubspec.yaml

```diff
@@ -27,14 +27,14 @@ dependencies:
   flutter_bloc: ^8.1.3
   get_it: ^7.6.0
   injectable: ^2.1.2
-  mobile_scanner: ">=3.5.1 <3.5.6"
-  device_info_plus: ^8.2.2
+  mobile_scanner: ^4.0.1
```

Comment:
❓ by increasing the dependency versions do we also have to up the required minimum flutter version as well? See line 11.

Reply:
That will be done in another commit. 
Switching to newer Dart SDK will allow to update other dependencies as well...